### PR TITLE
[Robot] Robot test for Service Schedule wizard Screen1 validations

### PR DESCRIPTION
**Pull request by kselvocki**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
- [ ] Any net new LWC work has Sa11y tests & 50% or above lines JEST test coverage  
- [ ] CRUD/FLS is enforced in Apex
- [ ] Permission sets are updated to account for CRUD|FLS|Tab|Classes
- [ ] Field sets are updated to account for new fields
- [ ] UX approval or UX not necessary
- [ ] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: W-0000000: Work Name (https://github.com/SalesforceFoundation/PMM/pull/303)
- [ ] All **acceptance criteria** have been met
    - [ ] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [ ] PR contains draft release notes
- [ ] QE story level testing completed

### CODE REVIEW

File: robot/pmm/tests/browser/ServiceSchedule/service_schedule_screen1.robot [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Resource       robot/pmm/resources/pmm.robot
+Library        cumulusci.robotframework.PageObjects
+...            robot/pmm/resources/ServiceSchedulePageObject.py
+...            robot/pmm/resources/ServicePageObject.py
+Suite Setup     Run Keywords
+...             Open Test Browser
+...             Setup Test Data
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
```
Comment (rmonica08) [inline robot/pmm/tests/browser/ServiceSchedule/service_schedule_screen1.robot:11]: Could you update the suite setup to be able to run the test as non admin users?

Suite Setup     Run Keywords
...             Open test browser            useralias=${test_user}             AND
...             Setup Test Data
Suite Teardown  Capture Screenshot and Delete Records and Close Browser

*** Variables ***
${test_user}             UUser

*** Keywords ***

File: robot/pmm/tests/browser/ServiceSchedule/service_schedule_screen1.robot [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    Set suite variable              ${service_schedule_name}
+    ${today} =                      Get Current Date                    result_format=%-m/%-d/%Y
+    Set suite variable              ${today}
+    ${earlier_date} =               Get Current Date                    result_format=%-m/%-d/%Y            increment=-5 days
+    Set suite variable              ${earlier_date}
+
+*** Test Cases ***
+Validate data on Screen1
+    [Documentation]                        Validates that start date cannot be after end date
+    ...                                    Validates that if Frequency=Daily/Weekly/Monthly, a session end time or session end after is set
```
Comment (rmonica08) [inline robot/pmm/tests/browser/ServiceSchedule/service_schedule_screen1.robot:31]: This needs to be removed, I think we will cover this scenario in a separate test

**Review by rmonica08 — COMMENTED** [review]
Looks good Kat,  just few updates to suite set up and documentation.
You could also update the keyword file when you commit this change.

File: robot/pmm/tests/browser/ServiceSchedule/service_schedule_screen1.robot (hunk 2 above)
Comment (kselvocki) [reply robot/pmm/tests/browser/ServiceSchedule/service_schedule_screen1.robot:31]: @rmonica08 - Do you think I should write it as a separate test? It is already included here.

File: robot/pmm/tests/browser/ServiceSchedule/service_schedule_screen1.robot (hunk 1 above)
Comment (kselvocki) [reply robot/pmm/tests/browser/ServiceSchedule/service_schedule_screen1.robot:11]: Thank you, adding this!

File: robot/pmm/tests/browser/ServiceSchedule/service_schedule_screen1.robot (hunk 2 above)
Comment (rmonica08) [reply robot/pmm/tests/browser/ServiceSchedule/service_schedule_screen1.robot:31]: Sorry, got a bit confused. We can leave it in the same test

File: robot/pmm/tests/browser/ServiceSchedule/service_schedule_screen1.robot (hunk 1 above)
Comment (kselvocki) [reply robot/pmm/tests/browser/ServiceSchedule/service_schedule_screen1.robot]: @rmonica08 - Added those lines and the updated keyword file.

**Review by rmonica08 — APPROVED** [review]

**Review by davidmreed — APPROVED** [review]

**Pull request merged**